packages/gatsby/src/redux/__tests__/__snapshots__/index.js.snap

```diff
@@ -71,7 +71,7 @@ Object {
     },
     "trackedQueries": Map {
       "/my-sweet-new-page/" => Object {
-        "dirty": 1,
+        "dirty": 8,
```

Comment:
Need to double check if this is safe - updating `gatsby` doesn't guarantee purging cache - so need to make sure this is safe to do as-is - or maybe flip the values for `FLAG_DIRTY_NEW_PAGE` / `FLAG_DIRTY_PAGE_CONTEXT` - or maybe even just add 2 completely new values - this needs some thinking about how updating gatsby will interact with cache on this one

Reply:
I checked and I think it's safe. We only compare against `0` before this change. But swapping values may still be a good thing to do purely for esthetic reasons, as `FLAG_DIRTY_NEW_PAGE` is the first flag value we set 😄

Reply:
Changed in https://github.com/gatsbyjs/gatsby/commit/01b10dde564fbc61fcb98fe35b3f5356b1926203